build/l10n.js

```diff
@@ -72,8 +72,10 @@
       document.documentElement.dataset.noCompleteBug = true;
     }
 
-    this.ctx.registerLocales(meta.defaultLanguage,
-                             Object.keys(meta.availableLanguages));
+    var al = meta.availableLanguages ?
```

Comment:
nit: I generally prefer longer "readable" variable names, especially if it's not a commonly used thing. It's more typing, but more readable imo. Would prefer ```langs``` or ```availableLangs```.